### PR TITLE
Add Math

### PR DESCRIPTION
**Pull request by rossant**
https://github.com/rossant/awesome-math

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -190,6 +190,7 @@ Please read the [contribution guidelines](contributing.md) or the [creating a li
 
 - [Papers We Love](https://github.com/papers-we-love/papers-we-love)
 - [Talks](https://github.com/JanVanRyswyck/awesome-talks)
+- [Mathematics](https://github.com/rossant/awesome-math)
```
Comment (sindresorhus) [inline readme.md]: Should be `Math`.

**Comment by sindresorhus** [comment]
Please read the [contributing guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [awesome manifest](https://github.com/sindresorhus/awesome/blob/master/awesome.md).

File: readme.md (hunk 1 above)
Comment (rossant) [inline readme.md]: I initially wrote "Math" then I read "check your spelling" in the contributing guidelines... ;) Just curious, which of the rules is not respected by typing "Mathematics"?

Comment (sindresorhus) [inline readme.md]: Your list is named `Math`. And the word `Math` is well-known enough.

**Comment by sindresorhus** [comment]
https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines

**Comment by rossant** [comment]
thanks
https://github.com/rossant/awesome-math/blob/master/contributing.md

**Pull request closed without merging**